src/services/identity/identity.ts

```diff
@@ -0,0 +1,275 @@
+import * as dotenv from 'dotenv';
+import type {
+	CredentialPayload,
+	DIDDocument,
+	DIDResolutionResult,
+	IIdentifier,
+	IVerifyResult,
+	ManagedKeyInfo,
+	TAgent,
+	VerifiableCredential,
+	VerifiablePresentation,
+} from '@veramo/core';
+import type {
+	ResourcePayload,
+	BulkRevocationResult,
+	BulkSuspensionResult,
+	BulkUnsuspensionResult,
+	CreateStatusList2021Result,
+	RevocationResult,
+	StatusCheckResult,
+	SuspensionResult,
+	UnsuspensionResult,
+} from '@cheqd/did-provider-cheqd';
+import type {
+	BroadCastStatusListOptions,
+	CheckStatusListOptions,
+	CreateStatusListOptions,
+	CredentialRequest,
+	StatusOptions,
+	UpdateStatusListOptions,
+	VeramoAgent,
+	VerificationOptions,
+} from '../../types/shared';
+import { Veramo } from './agent.js';
+
+dotenv.config();
+
+export interface IIdentity {
```

Comment:
Let's switch to `IIdentityService`.
```suggestion
export interface IIdentityService {
```